handlers/epochs.go

```diff
@@ -118,7 +118,7 @@ func Epochs(w http.ResponseWriter, r *http.Request) {
 		DisplayStart:    start,
 	}
 
-	if handleTemplateError(w, r, epochsTemplate.ExecuteTemplate(w, "layout", data)) != nil {
+	if handleTemplateError(w, r, "epochs.go / Done", epochsTemplate.ExecuteTemplate(w, "layout", data)) != nil {
```

Comment:
This looks like a copy and paste error and should be
`if handleTemplateError(w, r, "epochs.go / Epochs", epochsTemplate.ExecuteTemplate(w, "layout", data)) != nil {`
to be consistent with your other changes.